README.md

```diff
@@ -82,35 +82,35 @@ This is a partial implementation of https://www.swyx.io/the-surprisingly-high-ta
 
```

Comment:
I was also wondering if it would make sense to separate the deployment into a separate step, I think it might make it easier to understand in case you're just checking it out casually.

Reply:
maybe haha. really relying on users knowing how to deploy stuff